src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -66,6 +66,7 @@
 
     <muxc:NavigationView x:Name="SettingsNav"
                          Background="{ThemeResource SettingsPageBackground}"
+                         BackRequested="BackButton_Click"
                          IsBackButtonVisible="Collapsed"

```

Comment:
Hmm... Noticed that the back button isn't always visible. Can we change `IsBackButtonVisible` to `True` and just enable/disable it if there's something to go back to?